node/src/components/consensus/tests/consensus_des_testing.rs

```diff
@@ -148,7 +148,7 @@ where
 
 pub(crate) enum DeliverySchedule {
     AtInstant(Timestamp),
-    #[allow(dead_code)] // TODO: Use or remove this.
+    #[allow(dead_code)] // Drop variant used in tests.
```

Comment:
Isn't the whole module `#[cfg(test)]`? It shouldn't be complaining about dead code if it's actually used in tests then.

Reply:
It does indeed complain, I tried to resolve it